publisher/presenter/html/handlers.go

```diff
@@ -70,12 +83,14 @@ func (h *handlers) ContainerIndex(c echo.Context) error {
 		Title      string
 		Namespace  string
 		Containers []string
+		Total      uint64
 	}
 
 	return c.Render(http.StatusOK, "container-list.html", &data{
 		Title:      fmt.Sprintf("Container index (%s)", namespace),
 		Namespace:  namespace,
 		Containers: containers,
+		Total:      total,
```

Comment:
This parameter is for templates where pagination is not implemented yet, so publisher will not display "next page" link